packages/wrangler/src/pipelines/cli/list.ts

```diff
@@ -0,0 +1,28 @@
+import { readConfig } from "../../config";
+import { logger } from "../../logger";
+import * as metrics from "../../metrics";
+import { requireAuth } from "../../user";
+import { listPipelines } from "../client";
+import type { CommonYargsOptions } from "../../yargs-types";
+import type { ArgumentsCamelCase } from "yargs";
+
+export async function listPipelinesHandler(
```

Comment:
shouldn't this fn call `await printWranglerBanner();` as well?

Reply:
I am not sure, I copied it directly from https://github.com/cloudflare/workers-sdk/pull/7769/files#diff-bd3457733e5c7c3cdc0ea531bc35fc5858ec21ea84e1f65a11bee0f222efd727L330-L347.

Should all commands call `printWranglerBanner`?

Reply:
Added it.